src/main/java/cloud/fogbow/ras/core/models/orders/Order.java

```diff
@@ -208,4 +262,19 @@ public String toString() {
     public abstract ResourceType getType();
 
     public abstract String getSpec();
+
+    private class SerializableSystemUser {
+
+        private String className;
+        private String payload;
+
+        public SerializableSystemUser(SystemUser user) {
+            this.className = user.getClass().getName();
+            this.payload = GsonHolder.getInstance().toJson(user);
```

Comment:
Perhaps we should add a check here to see if the payload will fit in the database column (currently 1024 bytes - see previous comment), and throw an exception. Alternatively, we could do that at the SystemUser constructor. I am not sure, what would be the best choice, but this needs to be taken into account.